pages/glossary.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <h1 class="text-center text-2xl md:text-3xl lg:text-4xl mb-6">Glossary</h1>
+    <p class="text-center mb-3">
+      This glossary contains unusual or specialist words used across this website, including in filters and explanations, to help you better understand the terms and their meanings.
+    </p>
+    <div class="flex justify-center mb-5">
+      <button
+        @click="setSortMode('letters')"
+        :class="['px-4 py-2 mx-2', { 'bg-gray-200': sortMode === 'letters' }]"
+        :style="sortMode === 'letters' ? { outline: '1.5px solid black', fontWeight:'600'} : {}"
+        class="bg-white border rounded"
+      >
+        Sort Alphabetically
+      </button>
+      <button
+        @click="setSortMode('category')"
+        :class="['px-4 py-2 mx-2', { 'bg-gray-200 ': sortMode === 'category' }]"
+        :style="sortMode === 'category' ? { outline: '1.5px solid black', fontWeight:'600'} : {}"
+        class="bg-white border rounded"
+      >
+        Sort by Category
+      </button>
+    </div>
+    <div :style="{ marginLeft: '10%', marginRight: '10%' }">
+      <AlphabetGlossary v-if="sortMode === 'letters'" :glossary="glossary" />
+      <CategoryGlossary v-if="sortMode === 'category'" :glossary="glossary" :initialCategory="initialCategory" />
+    </div>
+  </div>
+</template>
+
+<script>
+import AlphabetGlossary from '@/components/AlphabetGlossary.vue';
```

Comment:
This is where I show off my lack of knowledge with Vue. Should we have the imports here or at the top of the file? I'm not sure what is convention. Presuming here because they will then sit within the script tag but thought I'd best check. Thanks Ruby